src/models/tools/image/image-import-export.ts

```diff
@@ -1,41 +1,50 @@
-import { createChange, ImageToolChange } from "./image-change";
-import { ITileExportOptions } from "../tool-content-info";
 import { safeJsonParse } from "../../../utilities/js-utils";
+import { ITileExportOptions } from "../tool-content-info";
 
-export interface IImageTileImportSpec {
+export interface ILegacyImageTileImport {
   type: "Image";
   url: string;
+  changes: string[];
 }
 
-export const isImageTileImportSpec = (snapshot: any): snapshot is IImageTileImportSpec =>
-              (snapshot?.type === "Image") && (snapshot.url != null) && !snapshot.changes;
+export const isLegacyImageTileImport = (snapshot: any): snapshot is ILegacyImageTileImport => {
+  return !!((snapshot?.type === "Image") && snapshot.changes);
```

Comment:
```suggestion
  return (snapshot?.type === "Image") && !!snapshot.changes;
```
Same result but the code's a little clearer.